.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,206 @@
+Hi, thank you for your interest in contributing to the WordPress Theme Review Coding Standards! We look forward to working with you.
+
+# Reporting Bugs
+
+Before reporting a bug, you should check what sniff an error is coming from.
+Running `phpcs` with the `-s` flag will show the name of the sniff with each error.
+
+Bug reports containing a minimal code sample which can be used to reproduce the issue are highly appreciated as those are most easily actionable.
+
+# Contributing patches and new features
+
+## Branches
+
+Ongoing development will be done in the `develop` branch with merges done into `master` once considered stable.
+
+To contribute an improvement to this project, fork the repo and open a pull request to the `develop` branch. Alternatively, if you have push access to this repo, create a feature branch prefixed by `feature/` and then open an intra-repo PR from that branch to `develop`.
+
+Once a commit is made to `develop`, a PR should be opened from `develop` into `master` and named "Next release". This PR will provide collaborators with a forum to discuss the upcoming stable release.
+
+# Considerations when writing sniffs
+
+## Public properties
+
+When writing sniffs, always remember that any `public` sniff property can be overruled via a custom ruleset by the end-user.
+Only make a property `public` if that is the intended behaviour.
+
+When you introduce new `public` sniff properties, or your sniff extends a class from which you inherit a `public` property, please don't forget to update the [public properties wiki page](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties) with the relevant details once your PR has been merged into the `develop` branch.
```

Comment:
If this is (or will be) written in the WPCS repo, maybe it wouldn't be a bad idea to just link to the part in the WPCS repos contributing template? Something like:

```
If you want to know a bit more about writing sniffs, check out the link in the wpcs repository with the detailed instructions.

[Link to instructions](link)
```

Reply:
Oh I get it. This should be moved down where the writing the sniff is, and we could put this in the `<details>` tags so that it can be expanded.

We can also then use this part in the readme (collapsed)

Reply:
Yeah, looking at it this paragraph makes no sense from the WPThemeReview point of view. We don't have a wiki, and we can document any changes when adding new sniff in the changelog. And maybe add a wiki down the line.